src/client/src/shared/helpers/password-toggle.ts

```diff
@@ -0,0 +1,29 @@
+import { dom } from "../libs/modules/dom/dom";
+
+const initPasswordToggle = () => {
```

Comment:
file name should match the name of the function
rename it to the init-password-toggle.ts